ui/window.go

```diff
@@ -2186,18 +2186,17 @@ func (window *Window) toggleUserContainer() {
 	window.RefreshLayout()
 }
 
-// toggleBareChat will display only the chatview as the fullscreen application
-// root. Calling this method again will revert the view to it's normal state.
+// toggleBareChat will display only the chatview and messageInput.
+// Calling this method again will revert the view to it's normal state.
 func (window *Window) toggleBareChat() {
 	window.bareChat = !window.bareChat
+	window.chatView.internalTextView.SetBorder(!window.bareChat)
+	window.leftArea.SetVisible(!window.bareChat)
```

Comment:
This won't suffice and isn't future proof. Instead, there should be a new container that groups up the two needed components (view and input). Put this into currentContainer, so nothing breaks. With this code for example, you forgot both the commandview and the usercontainer.

Reply:
Do you mean something other than the userList (line 2195)?

Also I thought it was handy to have the command view available while in bare mode.

Reply:
I see where I need a check for `config.Current.ShowUserContainer` and will add that in, but I think having the command view available is helpful, and we shouldn't need to create another grouped container - I think bare mode should focus on hiding elements that are not needed for chat or controlling cordless itself